cmd/rook/ceph/operator.go

```diff
@@ -60,16 +60,16 @@ func startOperator(cmd *cobra.Command, args []string) error {
 		rook.TerminateFatal(errors.Errorf("rook operator namespace is not provided. expose it via downward API in the rook operator manifest file using environment variable %q", k8sutil.PodNamespaceEnvVar))
 	}
 
-	rook.CheckOperatorResources(context.Clientset)
-	rookImage := rook.GetOperatorImage(context.Clientset, containerName)
+	rook.CheckOperatorResources(cmd.Context(), context.Clientset)
```

Comment:
I'm not sure how this context gets canceled or if it ever does. We never initialize it, cobra does if nil. I'm just wondering how useful that is but that's not a problem I believe, at least for code homogeneity.